model/config.go

```diff
@@ -2194,17 +2194,23 @@ type PluginState struct {
 	Enable bool
 }
 
+type PublicKeyDescription struct {
```

Comment:
Do we need this struct? Could we do with just a `[]string` instead?

Reply:
We sure can, but in future we might want to expand this struct to include additional fields. There is a same issue with the PluginState struct, isn't it?

Reply:
That's true: the only concern here is that this struct makes it much harder to "hand-edit" the config.json. Maybe it doesn't make sense to do that in the first place, but I wouldn't be surprised to get pushback from customers here.

If we simplified the model, is there a clear path in the future to deprecating the old model and adopting a new struct (e.g. honour both, and rewrite config on migrate?) Wondering if we can simplify now and introduce complexity later, if needed.

(Generally agree re: `PluginState` -- and in retrospect, it seems like the decision to make it a struct might have been premature.)

Reply:
I’m 2/5 on keeping the struct as is. Here is my rational:
1. "Hand-editing” this part of config does not make sense, especially in HA environment
2. We will have to expand this struct when we will have different types of signatures.
3. Deprecating old model is always a headache.

Reply:
Gotta agree with @lieut-data on this one. Not a huge fan of `PluginStates            map[string]*PluginState` either :). 

Even if we were to keep this struct, adding new fields here might still be a headache (if there is migration involved).

If we actually did have a need to store additional information, I'd be all for this, since we don't, my recommendation would be to use `[]string` here instead. My rationale is simplicity than anything. I say we go with this for mvp. 2.5/5 on this one 😁.

Reply:
2.5 > 2 😄

Reply:
:D